test/prog/dftb+/input/caffeine_xyz/dftb_in.hsd

```diff
@@ -27,9 +27,7 @@ Analysis {
   CalculateForces = Yes
 }
 
-ParserOptions {
-  ParserVersion = 8
-}
+InputVersion = "19.1"
```

Comment:
8 == 20.1 ?

Reply:
You're right, but than this would only test the reader and not the conversion, rolling the version back to "19.1" allows to test both (including the unsetting of the `InputVersion` node and the creation of the new `ParserVersion` node).

Reply:
Will that input work with the 19.1 code though (isn't the xyz format read post-release)? Probably better to have an example using features from that release.

Reply:
Added a new test for this case using `genFormat`, but this input will probably not work with version "18.2" because it is using the `InputVersion` node :wink:.